### PR TITLE
feat: [CDS-41562]: updating css for steps help panel and adding new help panel for more steps

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
### Summary

new button changs for steps help panel , also adding new panels for few more steps 

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 15 Issues 
* Code issue 15 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10569)

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.module.scss [hunk 1]
```diff
@@ -26,6 +26,16 @@
...
 }
+.floatingButton {
+  position: fixed;
+  bottom: 10px;
+  margin-left: 520px;
+}
+.floatingButtonClicked {
+  position: fixed;
+  bottom: 10px;
+  margin-left: 970px;
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.module.scss:37]: will this margin left work on bigger screens?

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.module.scss [hunk 2]
```diff
@@ -26,6 +26,16 @@
 .helpPanelStyleClose {
   width: 0px;
 }
+.floatingButton {
+  position: fixed;
+  bottom: 10px;
+  margin-left: 520px;
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.module.scss:32]: will this margin left work on bigger screens?

**Review by ishantmehtaharness — DISMISSED** [review]
Approved, with comments

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

**Review by PratyushGar — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**